.gitpod.yml

```diff
@@ -0,0 +1,12 @@
+tasks:
+  - before: |
+      sudo apt update -y
+      DEBIAN_FRONTEND=noninteractive sudo apt-get install -y bison python3-dev libcppunit-dev build-essential cmake autoconf tcl pkg-config git curl patch libtool-bin
```

Comment:
`bison, libcppunit-dev, tcl, pkg-config, patch, libtool-bin` also not required if just download pre-compiled thirdparty